### DISCUSSION TITLE
can't validate, AGV8: schema has a problem!

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-RC/community#294] [Issues]
Seeing this error when trying to save a Registry Entity creation form via UI. 

"can't validate, XXv7: schema has a problem!"

Any idea when this comes ?

Also is it possible to get more verbose feedback there ?

**Comment by parthlawate** [comment]
Update - this was sorted by updating UI to match the registry schema. However i would still request that we have more verbose feedback on the details of the error

**Reply by tejash-jl** [reply]
@parthlawate can you share more info on what was the issue?

**Reply by parthlawate** [reply]
UI Schema was not providing inputs expected by backend. .Once the missing field was added it got fixed. 2 things i think we can do here to make things simpler. @code-blooded-guy request you to specify the exact fix that was done

1. Give more verbose messages in the API response
2. Autogenerate the Form Schema based on backend entity schema  - @Renuka-S pls add this to the backlog
